packages/cdktf/lib/terraform-stack.ts

```diff
@@ -2,8 +2,6 @@ import { Construct, IConstruct, ISynthesisSession, Node } from "constructs";
 import { resolve } from "./_tokens";
 import * as fs from "fs";
 import * as path from "path";
-import camelcase = require("camelcase"); // ES module interop
```

Comment:
We can remove this from the bundledDependencies array in the package.json then

Reply:
Good point - done 👍